README.md

```diff
@@ -5,7 +5,7 @@
 Due to some nifty missing entitlement issues in the iOS Simulator, the Xcode 15 release has some **very** nasty CPU issues when running unit or UI tests. Until Apple releases an update addressing this issue, CI testing times can sometimes be up to two or **three** times longer than Xcode 14 running the same tests.
 
 ## What are the processes?
-During my testing, I found that a handful of *Poster processes (AegirPoster, ExtragalacticPoster, KaleidoscopePoster, etc.) are missing an entitlement, resulting in SpringBoard having a Very Bad Day™ every time it tries to talk to them. SpringBoard *really* likes talking to the Poster processes, which wastes a CPU core or two. `apsd` also has issues because it believes it needs to activate itself (I could be mistaken, but I don't think that the Simulator can activate itself with APNS). For folks that need to test push notifications with the Simulator, I've included a build that will stop `apsd` and a build that will let it burn.
+During my testing, I found that a handful of *Poster processes (AegirPoster, ExtragalacticPoster, KaleidoscopePoster, etc.) are missing an entitlement, resulting in SpringBoard having a Very Bad Day™ every time it tries to talk to them. SpringBoard *really* likes talking to the Poster processes, which wastes a CPU core or two. `apsd` also has issues because it believes it needs to activate itself (I could be mistaken, but I don't think that the Simulator can activate itself with APNS). By default, yeetd will kill `apsd` in the Simulator. For folks that need to test push notifications with the Simulator, just `defaults write yeetd killapsd false` to tell it not to kill apsd.
```

Comment:
I'm not familiar enough with userdefaults on macos to know if this is correct. I would have expected it to be `dev.biscuit.yeetd`, it's possible this was just bc I was running it from xcode